src/inc/eetwain.h

```diff
@@ -161,7 +161,7 @@ enum
 };
 
 #ifndef DACCESS_COMPILE
-
+#ifdef _TARGET_X86_
```

Comment:
@jkotas The existing implementation uses ``_TARGET_X86_``, and thus I uses ``_TARGET_X86_`` here. Is this related with ``WIN64EXCEPTIONS``?? If so, it would be better to use ``WIN64EXCEPTIONS`` as you mentioned.